scvi/inference/jvae_trainer.py

```diff
@@ -120,7 +128,7 @@ def data_loaders_loop(self):
         posteriors = [self._posteriors[name] for name in self.posteriors_loop]
         # find the largest dataset to cycle over the others
         largest = np.argmax(
-            [posterior.gene_dataset.X.shape[0] for posterior in posteriors]
+            [posterior.gene_dataset.n_cells for posterior in posteriors]
```

Comment:
I think we would like to remove gene_dataset from the whole codebase! adata is GREAT! so let's keep things uniform. 
Also, let's make the default kappa here to be 5 (see my github issue about hyperparameters for gimVI)